HotelBooking/HotelBooking.DataAccessLayer/Entities/RoomType.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+
+public enum RoomType
+{
+	Apartment,
+	PrivateBathroom,
+	SharedBathroom
+}
```

Comment:
Uwaga ogólna:
Zauważ, że wszędzie dajemy publiczny set. Oznacza to, że wartość z zewnątrz można dowolnie zmienić.
Zależy od podejścia, generalnie nie jest to dobre, z drugiej w wielu apkach widziałem takie rozwiązanie w tych klasach
reprezentujących tabelki. Narazie zostawiamy jak jest, ale warto o tym pamiętać, możemy sobie o tym też pogadać jeszcze